recipes/mender-client/debian-3.0.x/mender-client.dirs

```diff
@@ -0,0 +1,5 @@
+/etc/mender
```

Comment:
Q: Are really all of these needed?

Reply:
Probably not. Let's do a full review and clean-up in [QA-330](https://tracker.mender.io/browse/QA-330) :+1: